server/svix-server/src/worker.rs

```diff
@@ -326,6 +330,33 @@ pub async fn worker_loop(
     }
 }
 
+/// Nullifies the payload column for expired messages
+async fn clean_expired_messages(
+    pool: &DatabaseConnection,
+) -> std::result::Result<UpdateResult, DbErr> {
+    message::Entity::update_many()
+        .col_expr(message::Column::Payload, Expr::value(Value::Json(None)))
+        .filter(message::Column::Expiration.lte(Utc::now()))
+        .filter(Expr::col(message::Column::Payload).is_not_null())
+        .exec(pool)
+        .await
+}
+
+/// Runs every 5 minutes
+pub async fn expired_message_cleaner_loop(pool: &DatabaseConnection) -> Result<()> {
```

Comment:
```suggestion
pub async fn expired_message_cleaner_loop(pool: &DatabaseConnection) -> Result<!> {
```
I think this is what you want and then you can remove the `Ok(())`, as this function should never return a truthy value.